azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -265,8 +264,8 @@ Resume-ClusterNode
 ```
 
 > [!NOTE]
-> To apply the Network ATC settings across the cluster, repeat steps 1 through 5, step 7, and step 9 for each node of the cluster.
+> To apply the Network ATC settings across the cluster, repeat steps 1 through 5 (skip deleting the virtual switch as it was renamed), step 7, and step 9 for each node of the cluster.
 
 ## Next step
 
-Learn how to [Assess solution upgrade readiness for Azure Stack HCI](./validate-solution-upgrade-readiness.md).
\ No newline at end of file
+Learn how to [Assess solution upgrade readiness for Azure Stack HCI](./validate-solution-upgrade-readiness.md).
```

Comment:
@helmste - I don't see what has been changed here unless the text was deleted and added back.

Reply:
@ronmiab - I didn't change anything in this step, so I, too, was curious why this showed up as a 'new' line.